relayer-cli/src/commands/misbehaviour.rs

```diff
@@ -98,9 +99,15 @@ fn misbehaviour_handling<Chain: ChainHandle>(
     client_id: ClientId,
     update: Option<UpdateClient>,
 ) -> Result<(), Box<dyn std::error::Error>> {
-    let client_state = chain
-        .query_client_state(&client_id, Height::zero())
-        .map_err(|e| format!("could not query client state for {}: {}", client_id, e))?;
+    let client_state = {
+        let request = QueryClientStateRequest {
+            client_id: client_id.clone(),
+            height: Height::zero(),
+        };
+        chain
+            .query_client_state(request)
+            .map_err(|e| format!("could not query client state for {}: {}", client_id, e))?
+    };
```

Comment:
Rather than introduce a scoped block here and elsewhere can we instead just inline the request constructor in the method call to save on vertical/horizontal space? eg.

```suggestion
    let client_state = chain
        .query_client_state(QueryClientStateRequest {
          client_id: client_id.clone(),
          height: Height::zero(),
        })
        .map_err(|e| format!("could not query client state for {}: {}", client_id, e))?;
```

Reply:
Can we please do this everywhere where it saves space?

Reply:
Awesome, thank you and sorry for the churn.